### PR TITLE
Feat: Product page added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
1. first i add banner
2 second i add category section
3 third  i add product section

-product page related CSS you find in product.css file.
-product card component i use it from my component library.

**screenshots**
img-5
img-6

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
good to merge

**Review by paarthbajaj — APPROVED** [review]
Well written. You can merge.

File: product.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    color: var(--light-grey-color);
+}
+  
+.category .box-container .box:hover h3 {
+    color: var(--complementary-color);
+}
+  
+/* .products{
+    margin: 4rem;
+} */
```
Comment (t007rushi) [inline product.css:70]: remove comments

File: product.html [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+                        <p class="card-price-cut">₹349</p>
+                        <p class="card-percent-discount">5% off</p>
+                    </div>
+                    <a href="#">
+                        <button class="btn primary-btn"><i class="fas fa-shopping-cart"></i>Add to cart</button>
+                    </a>
+                </div>
+            </div>
+
+            <div class="box">
```
Comment (t007rushi) [inline product.html:156]: what signifies box? here you can have card-box/product-box for clearer mean

File: product.html [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+                <div class="product-card">
+                    <span class="card-badge-green">Best Seller</span>
```
Comment (t007rushi) [inline product.html:158]: this badge can be have @top-left or top-right & not middle

**Review by t007rushi — COMMENTED** [review]
Host on netlify

File: product.css (hunk 1 above)
Comment (kotalwar-krishna) [reply product.css:70]: ok I will do that

File: product.html (hunk 2 above)
Comment (kotalwar-krishna) [reply product.html:156]: ok I will make this change. for clear meaning

File: product.html (hunk 3 above)
Comment (kotalwar-krishna) [reply product.html:158]: I just Pull it card from the component library. I miss it I will do.

**Pull request merged**